card/mod/machines/lib/javascript/decko/doubleclick.js.coffee

```diff
@@ -0,0 +1,27 @@
+doubleClickActiveMap = { off: false, on: true, signed_in: decko.currentUserId }
+
+doubleClickActive = () ->
+  doubleClickActiveMap[decko.doubleClick]
+  # else alert "illegal configuration: " + decko.doubleClick
+
+doubleClickApplies = (el) ->
+  return false if ['.nodblclick', '.d0-card-header', '.card-editor', '.bridge-sidebar'].some (klass) ->
+    el.closest(klass)[0]
+    # double click inactive inside header, editor, or tag with "nodblclick" class
+  slot = el.slot()
+  return false if slot.find('.card-editor')[0]
+  # false if there is a card-editor open inside slot
+  slot.data 'cardId'
+
+triggerDoubleClickEditingOn = (el)->
+  slot = el.slot()
+  edit_view = decko.slotEditView(slot)
+  url = decko.path("~#{slot.data('cardId')}?view=#{edit_view}")
+  slot.reloadSlot url
```

Comment:
here is the new edit mode handling for double clicks. I got rid of code duplication for ajax calls by using `reloadSlot`

Reply:
ooh, I like this.